### ISSUE TITLE
Invalid output for deletion / insertion conflict

### ISSUE THREAD

**Issue by wetneb** [bmwill/diffy#28]
Consider the following merge scenario:

### Ancestor

```
{
    int a = 2;
}
```

### Ours
```
{
}
```

### Theirs
```
{
    int a = 2;
    int b = 3;
}
```

### Current diffy merge output
Successful merge without any conflict:
```
{
}
```

### Expected merge output
Conflicting merge:
```
{
<<<<<<< ours
||||||| ancestor
    int a = 2;
=======
    int a = 2;
    int b = 3;
>>>>>>> theirs
}
```

One can check that `git merge-file -p /tmp/ours /tmp/ancestor /tmp/theirs` indeed produces the output above.

**Comment by bmwill** [comment]
Thanks for the bug report! I'll try to dig into the root cause.

**Comment by bmwill** [comment]
Fixed by #30
